frontend/src/pages/index.tsx

```diff
@@ -14,7 +14,7 @@ const Index: NextPage = (): JSX.Element => {
 
   const { login, register } = useAuth({
     middleware: 'guest',
-    redirectIfAuthenticated: '/profile',
```

Comment:
no more profile page?

Reply:
This will redirect to the `/dashboard` page which is the default redirection 
![image](https://user-images.githubusercontent.com/108642414/185293571-225770bc-5052-4444-a2d8-a37d62bebf20.png)